ClassySchedule/src/main/java/samples/Main.java

```diff
@@ -40,10 +40,11 @@ public void start(Stage primaryStage) throws Exception{
         EventHandler<ActionEvent> event = new EventHandler<ActionEvent>() {
             public void handle(ActionEvent e)
             {
-                login();
+                adminLogin();
```

Comment:
I think we should maybe split this class out into its own class. Main should launch the login page and then anything after that should be handled by controllers/other classes